### ISSUE TITLE
There seems to be something wrong in the loss function.

### ISSUE THREAD

**Issue by lefiniti** [Oldpan/Deep-Painterly-Harmonization-Pytorch#4]
There seems to be something wrong in the loss function.The results are very poor.
